app/models/spree/payment.rb

```diff
@@ -51,6 +51,7 @@ class Payment < ApplicationRecord
     scope :pending, -> { with_state('pending') }
     scope :failed, -> { with_state('failed') }
     scope :valid, -> { where.not(state: %w(failed invalid)) }
+    scope :void, -> { with_state('void') }
```

Comment:
The commit message could explain why we are doing this. You could also include that in the commit which needs this scope to demonstrate the use.

Reply:
Makes sense, thanks!